src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt

```diff
@@ -192,15 +192,27 @@ object EstimatedItemValueCalculator {
         } else {
             0.0
         }
-
-    private fun isUselessAttribute(internalName: String): Boolean {
-        if (internalName.contains("RESISTANCE")) return true
-        if (internalName.contains("SPEED")) return true
-        if (internalName.contains("EXPERIENCE")) return true
-        if (internalName.contains("FORTITUDE")) return true
-        if (internalName.contains("ENDER")) return true
-
-        return false
+    private val badAttributeList = listOf(
+        // armor
+        "ARACHNO_RESISTANCE",
+        "BLAZING_RESISTANCE",
+        "ENDER_RESISTANCE",
+        "EXPERIENCE",
+        "FORTITUDE",
+        "SPEED",
+        "UNDEAD_RESISTANCE",
```

Comment:
Maybe remove this as it is situationally useful (MP/UR Aurora)?

Reply:
Yeah, this is temporary until we get a "Good rolls" list added to Repo that I think Empa was working on for a feature